### DISCUSSION TITLE
Can I change language?

### DISCUSSION THREAD

**Discussion by sky172839465** [khmyznikov/pwa-install#78] [Q&A] [answered]
Hi, thanks for making this beautiful install dialog.
Is it possible set specific language from params or change language from DOM method?
I need to make sure install description always show in `zh` language, even user browser `navigator.language` is `en-US`

Can we public changeLocale to DOM method?
https://github.com/khmyznikov/pwa-install/blob/9ffcd04ff2c883242b33a48b3f1246f586754ad5/src/index.ts#L250

**Comment by khmyznikov** [comment, marked as answer]
[Already discussed this. ](https://github.com/khmyznikov/pwa-install/pull/43) Language of installation prompt should match the browser UI language.
